repository/git.go

```diff
@@ -16,15 +17,19 @@ import (
 )
 
 func cloneGitRepository(ctx context.Context, repo Repository, localPath string, options GitHubOptions) (*git.Repository, error) {
-	url := fmt.Sprintf("https://github.com/%s.git", repo.FullName())
+	gitUrl, err := url.JoinPath(options.Url, repo.GitFullName())
```

Comment:
url.JoinPath has been added in go 1.19, so to use it you'll need to update the version of go in go.mod (it's used automatically in the github actions to install the same version of go for the CI jobs)

Reply:
done